mlrun/utils/logger.py

```diff
@@ -93,6 +96,71 @@ def _record_with(self, record):
         return record_with
 
 
+class CustomFormatter(HumanReadableFormatter):
+    """
+    To enable custom logger formatter, configure MLRun with the following env variables:
+    1. "MLRUN_LOG_FORMATTER" = "custom" - change the default log formatter.
+    2. "MLRUN_CUSTOM_FORMAT" = "> {timestamp} [{level}] Running module: {module} {message} {more}" - logger format
+        * Note that your custom format must include those 4 fields - timestamp, level, message and more
+    If the custom format is not configured properly , MLRun will use the default logger (human format).
+    """
+
+    def format(self, record) -> str:
+        logger = create_logger("INFO", "human", "logger", sys.stdout)
```

Comment:
why basing the logger on human and not on what given by config (human_extendned / json / human)? also you're setting the logger to be info and ignoring the config as well